pkgs/development/tools/analysis/ikos/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub,
+  cmake, boost, gmp, llvmPackages_7, sqlite, python3,
+  ocamlPackages, mpfr, ppl, doxygen, graphviz }:
```

Comment:
Could you please update this so that it matches our standard style, as specified in the manual? 
https://nixos.org/nixpkgs/manual/

Reply:
I searched for "style" and didn't find anything relevant. I've changed it into a single line, which is what other packages in the same folder seem to have.

Reply:
I'll do that. Thanks for the review.